src/test/java/seedu/linkedout/logic/parser/SearchCommandParserTest.java

```diff
@@ -45,49 +54,86 @@ public void parse_keywordMissing_throwsParseException() {
         // missing job keyword
         assertParseFailure(parser, " j/", String.format(
                 MESSAGE_INVALID_COMMAND_FORMAT, SearchCommand.MESSAGE_CONSTRAINTS));
+
+        //missing skill keyword
+        assertParseFailure(parser, " s/", String.format(
+                MESSAGE_INVALID_COMMAND_FORMAT, SearchCommand.MESSAGE_CONSTRAINTS));
+
```

Comment:
Consider adding one case of " j/ s/" to illustrate multiple keywords missing, though if multiple prefix is use, first one without keyword will already trigger ParseFailure.

Reply:
Added!